crates/rpc-types-eth/src/lib.rs

```diff
@@ -53,5 +53,6 @@ pub use work::Work;
 
 /// This module provides implementations for EIP-4337.
 pub mod eip4337;
+pub use eip4337::{SendUserOperationResponse, UserOperation};
```

Comment:
please also rename this module and anything in it referencing EIP-4337 as this is not an EIP, it's an ERC